documentation/src/main/asciidoc/topics/con_clients_unmarshalled_objects.adoc

```diff
@@ -0,0 +1,14 @@
+[id='clients-unmarshalled-objects_{context}']
+= Clients and POJOs
+Even though {brandname} does not recommend doing so, clients can operate on caches that store unmarshalled POJOs with the `application/x-java-object` media type.
+
+[discrete]
+== Hot Rod clients
+
+Hot Rod client marshallers must be available to {brandname} Server so it can deserialize your Java objects.
+By default, the ProtoStream and Java Serialization marshallers are available on the server.
+
+[discrete]
+== REST clients
+
+REST clients must use either JSON or XML for keys and values so {brandname} can convert to and from POJOs.
```

Comment:
Maybe it's mentioned elsewhere, but we should say somewhere that the XML transcoder uses the same allow list that's used by `JavaSerializationMarshaller`.

Reply:
@danberindei How about adding this as a note: "{brandname} requires you to add Java classes to the deserialization allowlist to convert XML to and from POJOs."

Not sure if we need to mention `JavaSerializationMarshaller` under the REST client heading.